packages/renderer/src/docs/assay.vue

```diff
@@ -1,3 +1,181 @@
+<!-- TODO fix add buttons -->
+
 <template>
-  <h5>Assay Help</h5>
+  <div>
+  <h5>Assay</h5>
+
+    <div class="text-body2 text-justify">
+      In the Assay menu you can adapt the metadata of one assay added to your ARC.
+    </div>
+    <br>
```

Comment:
Please avoid using `<br>` instead define a class on the parent element which applies a margin-bottom to all children except the last.

```css
.parent-class > :not(:last-child) {
  margin-bottom: 1rem; /* Adjust the margin value as needed */
}
```

Reply:
or apply a margin to the child elements directly